code/game/objects/structures/signs/signs_warning.dm

```diff
@@ -1,122 +1,181 @@
+//warning signs
+
+
+///////DANGEROUS THINGS
+
 /obj/structure/sign/warning
-	name = "\improper WARNING"
+	name = "\improper WARNING sign"
+	sign_change_name = "Warning"
 	desc = "A warning sign."
 	icon_state = "securearea"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/securearea
-	name = "\improper SECURE AREA"
+	name = "\improper SECURE AREA sign"
+	sign_change_name = "Warning - Secure Area"
 	desc = "A warning sign which reads 'SECURE AREA'."
+	is_editable = TRUE
 
 /obj/structure/sign/warning/docking
-	name = "\improper KEEP CLEAR: DOCKING AREA"
+	name = "\improper KEEP CLEAR: DOCKING AREA sign"
+	sign_change_name = "Warning - Docking Area"
 	desc = "A warning sign which reads 'KEEP CLEAR OF DOCKING AREA'."
+	is_editable = TRUE
 
 /obj/structure/sign/warning/biohazard
-	name = "\improper BIOHAZARD"
+	name = "\improper BIOHAZARD sign"
+	sign_change_name = "Warning - Biohazard"
 	desc = "A warning sign which reads 'BIOHAZARD'."
 	icon_state = "bio"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/electricshock
-	name = "\improper HIGH VOLTAGE"
+	name = "\improper HIGH VOLTAGE sign"
+	sign_change_name = "Warning - High Voltage"
 	desc = "A warning sign which reads 'HIGH VOLTAGE'."
 	icon_state = "shock"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/vacuum
-	name = "\improper HARD VACUUM AHEAD"
+	name = "\improper HARD VACUUM AHEAD sign"
+	sign_change_name = "Warning - Hard Vacuum"
 	desc = "A warning sign which reads 'HARD VACUUM AHEAD'."
 	icon_state = "space"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/vacuum/external
-	name = "\improper EXTERNAL AIRLOCK"
+	name = "\improper EXTERNAL AIRLOCK sign"
+	sign_change_name = "Warning - External Airlock"
 	desc = "A warning sign which reads 'EXTERNAL AIRLOCK'."
 	layer = MOB_LAYER
+	is_editable = TRUE
 
 /obj/structure/sign/warning/deathsposal
-	name = "\improper DISPOSAL: LEADS TO SPACE"
+	name = "\improper DISPOSAL: LEADS TO SPACE sign"
+	sign_change_name = "Warning - Disposals: Leads to Space"
 	desc = "A warning sign which reads 'DISPOSAL: LEADS TO SPACE'."
 	icon_state = "deathsposal"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/bodysposal
-	name = "\improper DISPOSAL: LEADS TO MORGUE"
+	name = "\improper DISPOSAL: LEADS TO MORGUE sign"
+	sign_change_name = "Warning - Disposals: Leads to Morgue"
 	desc = "A warning sign which reads 'DISPOSAL: LEADS TO MORGUE'."
 	icon_state = "bodysposal"
-
-/obj/structure/sign/warning/pods
```

Comment:
you axed this by accident

Reply:
It's still there, it just got moved. A bunch of these types got moved around to categorize them more clearly.